data/hails.txt

```diff
@@ -1672,12 +1796,14 @@ phrase "friendly pirate"
 		"tried this"
 		"used this"
 	word
-		" new "
-		" crazy "
-		" amazing "
-		" fantastic "
-		" incredible "
-		" facinating "
+		" "
+	word
+		"new "
+		"crazy "
+		"amazing "
+		"fantastic "
+		"incredible "
+		"facinating "
```

Comment:
You messed up the formatting. All of these need leading spaces.

Reply:
I added the word " " like was requested of me to do earlier.

Reply:
Oh so you did, my mistake!